tests/app/v2/notifications/test_post_notifications.py

```diff
@@ -1156,10 +962,6 @@ def test_post_notification_with_document_upload(
         {"file": decoded_file, "filename": filename, "sending_method": sending_method},
     )
 
-    notification = Notification.query.one()
-    assert notification.status == NOTIFICATION_CREATED
-    assert notification.personalisation == {"document": document_response}
```

Comment:
hrm, this wouldn't be done before we put it into redis, but we should move this test to somewhere else.

Reply:
skipped